tests/CopyFilesTaskIntegrationTest.php

```diff
@@ -13,24 +14,28 @@ class CopyFilesTaskIntegrationTest extends \PHPUnit\Framework\TestCase
 
 	private const TEMP_DIRECTORY_PATH = __DIR__ . '/temp';
 
-	public function testCopyFile(): void
+	/**
+	 * @return mixed[][]|\Generator
+	 */
+	public function copyFileDataProvider(): Generator
```

Comment:
Since the original aim of having one target per test case is now gone and the targets were named like this only because they matched the test methods I think they should change now (and also right now `testCopyFile` is both a real test method name and a case here).

We can use normal `kebab-case` format (and without `test...`) as is common otherwise (both for targets and properties), but I would do it in a separate commit afterwards.

Reply:
Renamed the targets using `kebab-case`.